game.py

```diff
@@ -20,7 +20,24 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+    print("* LETS PLAY SNOWMAN ! *")
+    wrong_guesses_list = []
```

Comment:
Different development teams will have different guidelines, but we typically want to describe the contents of a variable and can leave off the data type in the name. This is especially true in languages other than Python that require folks to declare a variable's type when we create it. 

It would be completely valid and in some cases preferable to call this variable `wrong_guesses` rather than `wrong_guesses_list`.